include/arch/arm/aarch32/thread.h

```diff
@@ -111,11 +111,11 @@ struct _thread_arch {
 		uint32_t mode;
 
 #if defined(CONFIG_ARM_STORE_EXC_RETURN)
-		__packed struct {
```

Comment:
would it mean that this structure was not packed for the whole time?
if so (it was not needed) why it is packed at all?

Reply:
To be honest I did not observe the build issue before. I think our configurations did not enable this functionality previously.

Reply:
I think it would work even if the struct wasn't packed, but maybe it's not needed. We could do a BUILD_ASSERT instead.

Reply:
If this is not needed I would prefer not to have it packed as it has its side effects.
Since it is inside union with `uint32_t mode` and struct members add up to 32 -bit I doubt it will add any benefits for aligning or anything.

Reply:
`__packed` attribute was removed. Please rerevew